samples/wrapper/wrapper_config.h.in

```diff
@@ -0,0 +1,15 @@
+#ifndef BOINC_WRAPPER_CONFIG_H
```

Comment:
why you can't use original config.h file?
ok, I understand that it's generated during configure script run, but why it's required to create a separate file template for this and just not reuse existing one?